internal/manager/setup.go

```diff
@@ -58,7 +58,7 @@ func SetupLoggers(c *Config, output io.Writer) (logrus.FieldLogger, logr.Logger,
 
 func setupControllerOptions(logger logr.Logger, c *Config, dbmode string, featureGates map[string]bool) (ctrl.Options, error) {
 	logger.Info("building the manager runtime scheme and loading apis into the scheme")
-	scheme, err := scheme.Get(featureGates)
+	scheme, err := scheme.Get(featureGates, c.KnativeIngressEnabled)
```

Comment:
Did this change intend to start controller for `knative.Ingress` when it is enabled? This change should be included in another PR.